### DISCUSSION TITLE
Where can I find the source code for the demo app shown on the Textualize website?

### DISCUSSION THREAD

**Discussion by unwize** [Textualize/textual#439] [Q&A] [answered]
Here's the video in question: https://d32lig6y1jobn4.cloudfront.net/textual.mp4

And it is located here: https://www.textualize.io/

I am trying to create an animated side-bar *exactly* like the one shown in the video and would be immensely grateful for the code. 

Thanks for any assistance.

**Comment by willmcgugan** [comment, marked as answer]
Basic.py in the CSS branch, but bear in mind it is still a WIP

**Reply by unwize** [reply]
Thank you!
